Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -128,6 +128,30 @@ public extension Node where Context: HTMLLinkableContext {
     }
 }
 
+public extension Attribute where Context: HTMLLinkableContext {
+    /// Assign a hreflang string to this element.
+    /// - parameter hreflang: The hreflang of the link to assign.
+    static func hreflang(_ hreflang: String) -> Attribute {
```

Comment:
Shouldn't this accept a `Language` value, rather than a string?

Reply:
Done.  And added "x-default" to language enum.

Reply:
More long-term and for complete support here for sites or pages that need to have [localized language versions](https://support.google.com/webmasters/answer/189077?hl=en#language-codes) of a page indexed, country support ([ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)) should be added to hreflang as well. Which would allow for "en-GB" for instance.  I just pushed an update with a potential solution.  The biggest issue I see is the coutry names, many are multiple words, so I chose to just basically map lower-case country inputs to uppercase: `hreflang(.english, .us)` which renders `hreflang="en-US"`.

Reply:
ok, I added "auEnglish" and "caEnglish" (Australia/Canada) that probably covers the most common English countries, removed the "Country.swift" file, and updated the test.